apps/sms/views/conversation/js/conversation.js

```diff
@@ -2172,7 +2172,7 @@ var ConversationView = {
 
   sendMessage: function conv_sendMessage(opts) {
     var content = Compose.getContent(),
-        subject = Compose.getSubject(),
+        subject = Compose.isSubjectVisible ? Compose.getSubject() : null,
```

Comment:
Yeah, I think it's better, so we can remove my temporal workaround in https://github.com/mozilla-b2g/gaia/blob/2433ba5a10ede2f3f221376a4ed7fc3c33ad266d/apps/sms/views/conversation/js/conversation.js#L2914 :)